forms/FormField.php

```diff
@@ -138,6 +138,8 @@ public static function name_to_label($fieldName) {
 	 * @return string
 	 */
 	public static function create_tag($tag, $attributes, $content = null) {
+		Requirements::javascript(FRAMEWORK_DIR . '/javascript/FieldToggleDescription.js');
```

Comment:
I think if it's in LeftAndMain we should remove it from here. This is a CMS specific feature not a field specific one?